dist/powerbi-client.d.ts

```diff
@@ -87,8 +87,8 @@ declare module "embed" {
     import * as models from 'powerbi-models';
     global  {
         interface Document {
-            mozCancelFullScreen: Function;
-            msExitFullscreen: Function;
+            mozCancelFullScreen: any;
+            msExitFullscreen: any;
```

Comment:
This file is auto generated.
your changes will be overridden in the next submit.
The correct thing to do is to change the ts files in the src folder.

Reply:
https://github.com/microsoft/PowerBI-JavaScript/search?q=mozCancelFullScreen&unscoped_q=mozCancelFullScreen